integration-tests/actions/automationv2/actions.go

```diff
@@ -36,6 +38,8 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/chaintype"
 	"github.com/smartcontractkit/chainlink/v2/core/store/models"
 	"github.com/smartcontractkit/chainlink/v2/core/utils"
+
+	ctfTestEnv "github.com/smartcontractkit/chainlink-testing-framework/docker/test_env"
```

Comment:
This should be all lowercase e.g. ctftestenv

Reply:
Addressed this is next PR on this work.